native/libcst/src/nodes/statement.rs

```diff
@@ -2027,7 +2044,9 @@ impl<'a> Inflate<'a> for With<'a> {
 
         self.whitespace_after_with =
             parse_simple_whitespace(config, &mut (*self.with_tok).whitespace_after.borrow_mut())?;
+        self.lpar = self.lpar.map(|lpar| lpar.inflate(config)).transpose()?;
         self.items = self.items.inflate(config)?;
+        self.rpar = self.rpar.map(|rpar| rpar.inflate(config)).transpose()?;
```

Comment:
For other (parenthesized) comma separated lists, `rpar` owns the whitespace after a trailing comma. This implementation currently diverges because the item's `inflate` will consume the whitespace before `rpar` gets inflated. Check out `Tuple`'s `Inflate` implementation and the `inflate_element` associated function on `Element`

Reply:
I think I have this working correctly now, although it still needs a test case.

Although I do wish there were some docs on how the lower-level components of the inflation work - it seems like the 'config' has a mutable store of whitespace and inflate kind of vacuums up anything adjacent to a token that hasn't already been claimed? I'm hoping to read that code in more detail tomorrow, today I just focused on making it run.

Reply:
Yes, all of this code is in dire need of some documentation.